docs/_config.yml

```diff
@@ -1,10 +1,10 @@
 title: Insomnia Docs
 email: support@insomnia.rest
 description: >- # this means to ignore newlines until "baseurl:"
-  Insomnia Docs: Deliver high quality APIs through standards 
+  Insomnia Docs: Deliver high quality APIs through standards
   and collaboration with the Insomnia API design platform.
 baseurl: "/" # the subpath of your site, e.g. /blog
-url: "https://docs.insomnia.rest/" # the base hostname & protocol for your site, e.g. http://example.com
+url: "https://docs.insomnia.rest" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
The trailing slash here was causing any absolute URL to have double slashes in it, eg `https://docs.insomnia.rest//foobar`